src/content/app/tools/blast/state/blast-results/blastResultsSlice.ts

```diff
@@ -68,6 +68,7 @@ export type BlastSubmission = {
     status: JobStatus;
     data: null; // TODO: add data type
   }>;
+  isExpanded: boolean; // used to retain expanded / collapsed state in the jobs list view
```

Comment:
I don't think this piece of state belongs on a blast submission:

- It isn't clear what "isExpanded" means
- There will be lots of expandable/collapsible things related to a Blast submission on other pages
- All this UI fluff should either belong in its own reducer (either a dedicated reducer per each of the BLAST pages, or a single UI reducer for all the blast pages), or in a dedicated section of the blast results slice.